edubots_surveys.Rnw

```diff
@@ -956,29 +956,29 @@ corrplot(chisq$residuals, is.corr = FALSE)
     }
     \end{minipage}
 \end{center}
-    \caption{Analysis for gender and post-covid changes: left) Pearson's residuals for the Chi-square test; right) Gender*post-covid changes crosstabulation, including standardized residuals \cite{agresti2013categorical} (also adjusted standardized residuals) and contribution percentage to the total Chi-square test of each cell.}
+    \caption{Analysis for gender and post-COVID-19 changes: left) Pearson's residuals for the Chi-square test; right) Gender*post-COVID-19 changes crosstabulation, including standardized residuals \cite{agresti2013categorical} (also adjusted standardized residuals) and contribution percentage to the total Chi-square test of each cell.}
 \label{figure:gender*postcovid_chisq}
 \end{figure}
 
-As mentioned, the graphs on the left hand side represent the Pearson's
+As mentioned, the graphs on the left-hand side represent the Pearson's
 residuals, showing the difference between the observed and the expected values
 for each cell. Thus, large residuals indicate that variables are not truly
-independent. In our case, the blue shows positive contributions and red negative
-contributions to the test. Moreover, the saturation shows how large the
-contribution is in contrast to the expected value by chance. Additionally,
-tables on the right side show adjusted standardized residuals that according to
+independent. In our case, blue shows positive contributions and red negative
+contributions to the test. Additionally, the saturation shows how large the
+contribution is in contrast to the expected value by chance. 
+Tables on the right side show adjusted standardized residuals that according to
 \cite{agresti2013categorical}, if greater than +/-2 for cases with few cells,
-indicate lack of fit of $H_0$ (in boldface in our table). Complementary, the
+indicates lack of fit of $H_0$ (in boldface in our table). Complementarily, the
 table also shows the percentage of contribution to the test of each cell
-(highest percentage per column also in boldface).
+(the highest percentage per column also in boldface).
 
 Regarding gender in Fig. \ref{figure:gender*postcovid_chisq}, the results show a
 high positive contribution from female educators that did not use a messaging
 app before the COVID-19 pandemic but do use it now; approximately 31\% of the
 test results are explained by this cell. In contrast, male educators that
-respond the same were less than expected (negative contribution that explains
+responded the same were less than expected (negative contribution that explains
```

Comment:
"responded the same"? The same *answer*? In the same way?

Reply:
I'm going for "in the same way" in a new commit